calicoctl/commands/node/status.go

```diff
@@ -129,7 +136,19 @@ func drawTable(birdOut string) {
 
 		if s != "" {
 			col := []string{}
-			fields := strings.Fields(line)[3:6]
+			f := strings.Fields(line)[3:]
+			fields := make([]string, 3)
+			copy(fields, f[0:3])
+
+			// This is to include the extra info when the "Info" field is other than "Established"
+			// For example in case of "Mesh_fd80_24e2_f998_72d7__2 BGP  master   start  17:56:21 Active Socket: Connection closed"
+			// providing only "Active" in the Info section is not enough, so we append rest of the info into the last field
+			if len(f) > 3 {
+				for _, e := range f[3:] {
+					fields[2] = fields[2] + " " + e
+				}
+			}
+
 			if strings.HasPrefix(s, "Mesh_") {
 				s = s[5:]
```

Comment:
I called is `s` because it's not exactly an IP (it's like `Mesh_fd80_24e2_f998_72d7__2`) Anyway, makes sense to name it something else, so I changed it to `ipString`